### PR TITLE
Add integration test to forward models

### PR DESCRIPTION
**Pull request by oyvindeide**

### CODE REVIEW

File: tests/test_console_scripts.py [hunk 1]
```diff
@@ -22,3 +42,29 @@ def test_console_scripts_help(script_runner, entry_point):
...
     assert ret.success
+
+
+@pytest.mark.parametrize(
+    "forward_model",
+    [
+        ("OTS(<CONFIG>=config.ots)",),
+    ],
+)
+def test_console_script_intagration(setup_tmpdir, command):
```
Comment (berland) [inline tests/test_console_scripts.py:53]: typo: int*e*gration

Comment (oyvindeide) [reply tests/test_console_scripts.py]: Thanks :)

File: tests/test_ert_integration.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+def test_console_script_integration(
+    setup_tmpdir, forward_model, configuration, expected_error
+):
+    if (
+        forward_model in ["DESIGN2PARAMS", "GENDATA_RFT", "DESIGN_KW"]
+        and sys.platform == "darwin"
+    ):
+        pytest.skip(
+            (
+                "Forward models have same name as executable and "
```
Comment (jondequinor) [inline tests/test_ert_integration.py:49]: Maybe link to a bug?

**Review by jondequinor — APPROVED** [review]
LGTM

**Pull request merged**